test/unit/models/users.test.js

```diff
@@ -115,6 +115,16 @@ describe("users", function () {
       const { user } = await users.fetchUser({ githubUsername });
       expect(user).to.haveOwnProperty("github_created_at");
     });
+
+    it("it should filter out the id field while updating profileDiff", async function () {
```

Comment:
we should write test cases where the `devFeatureFlag` query parameter is missing or incorrectly set.